### PR TITLE
Creating a multi-page form using Umbraco Forms

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

Created a Tutorial on how to create a multi-page form using Umbraco Forms

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

v14

## Deadline (if relevant)

Anytime

### CODE REVIEW

**Comment by eshanrnh** [comment]
The video cannot be added in the article since it uses Macro (discontinued from v14) to embed the Form in the RTE.

File: 14/umbraco-forms/tutorials/creating-a-multipage-form.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+![Final Page Details](images/Final-page.png)
+
+## Embed the Form on a Web Page
+
+Once you are satisfied with your multi-page form, it is time to embed it on your website.
+
+To display the form on the website, follow these steps:
+
+1. Navigate to the **Content** section of Umbraco.
+2. Click **+** next to the parent page of the website.
```
Comment (sofietoft) [inline 14/umbraco-forms/tutorials/creating-a-multipage-form.md:124]: Is a step for picking a Document Type missing here?
Suggestion: Perhaps there should be a prerequisite to this tutorial, for having a Document Type with the Form Picker?

File: 14/umbraco-forms/tutorials/creating-a-multipage-form.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+```cs
+@await Component.InvokeAsync("RenderForm", new { formId = Guid.Parse("<form guid>"), theme = "default", includeScripts = false })
+```
+
+For more information, see the [Rendering Forms](../developer/rendering-forms.md) article.
+
+{% hint style="info" %}
+For Umbraco Forms to work correctly, you need to include some client dependencies. For more information, see the [Preparing Your Frontend](https://docs.umbraco.com/umbraco-forms/developer/prepping-frontend) article.
+{% endhint %}
```
Comment (sofietoft) [inline 14/umbraco-forms/tutorials/creating-a-multipage-form.md:147]: A suggestion for this entire section:

Start by directing the user to the Preparing your frontend article.
Then direct the user to the Rendering Forms article.

That way this section is slimmed down, and we do not end up repeating content that is already written in those articles.

What do you think about that idea? 🤔 

Side note: The Rendering Forms article need to be updated as well, as it still mentions Macros 😅 I'll add a task for it!

File: 14/umbraco-forms/tutorials/creating-a-multipage-form.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+## Testing the Form
+
+1. Go to the **Info** tab of the *Book a Meeting!* page.
```
Comment (sofietoft) [inline 14/umbraco-forms/tutorials/creating-a-multipage-form.md:151]: I believe it's called "Workspace View" and not tab/content app anymore 😅 

```suggestion
1. Go to the **Info** workspace view of the *Book a Meeting!* page.
```

**Review by sofietoft — COMMENTED** [review]
Great tutorial 💪 
I really like the "real life" example that you've chosen for the tutorial - makes it seem relevant!

I've added some suggestions here and there.
Also, a few additional thoughts:
- Would be great with some more descriptive image captions, especially for the images of the "entire pages" after all fields are added. These images are a little confusing as they're not introduced in the text.
- When introducing the condition - I think it would be great to explain the condition that's set. So, writing something like "Show this field if xx field is xx". Just to we do not rely only on the image/visuals for this.

Let me know if something doesn't make sense!

File: 14/umbraco-forms/tutorials/creating-a-multipage-form.md (hunk 1 above)
Comment (eshanrnh) [reply 14/umbraco-forms/tutorials/creating-a-multipage-form.md:124]: Oh no 🙈, Yes a step and an image is missing 😳. I like the idea of adding a Prerequisite section as this section expects users to have some existing content to embed the Form.

File: 14/umbraco-forms/tutorials/creating-a-multipage-form.md (hunk 2 above)
Comment (eshanrnh) [reply 14/umbraco-forms/tutorials/creating-a-multipage-form.md:147]: I like it. I've removed the text from the Note and placed it at the start.

Awesome. I noticed the Macro in the article but did not add a task to remove it 🙈

**Comment by eshanrnh** [comment]
Thanks @sofietoft for the review and suggestions 🙌I've incorporated all of it. Let me know how it looks😀

**Review by sofietoft — APPROVED** [review]
LGTM 💪 
Super tutorial - and shame about how to video material is outdated, but that's a task for another day 😅

**Pull request merged**